cacheops/invalidation.py

```diff
@@ -25,7 +33,8 @@ def invalidate_dict(model, obj_dict, using=DEFAULT_DB_ALIAS):
     prefix = get_prefix(_cond_dnfs=[(model._meta.db_table, list(obj_dict.items()))], dbs=[using])
     load_script('invalidate')(keys=[prefix], args=[
         model._meta.db_table,
-        json.dumps(obj_dict, default=str)
+        json.dumps(obj_dict, default=str),
+        conj_del_fn()
```

Comment:
Ah, you're calling this to memoize the function call, right?  Otherwise you're not doing anything with the return value.

Reply:
Well I'm passing it to the `invalidate` script!

Reply:
Oh, geez.  It looks like a statement on a line of its own, and the diff doesn't help figure that out, oy.

Reply:
`suor`, the upstream dev, wants me to change that anyway...